### PR TITLE
Incremental improvements for migration docs

### PR DESCRIPTION
**Pull request by svrnm**
Some incremental improvements for migration docs:

* Some introduction for where Otel comes from and why we provide oc & ot migration (copied some of your words @aabmass, hope that's OK :))
* Also some words about jaeger (copied from the jaeger docs (cc @pavolloffay) & @jpkrohling [blog post](https://medium.com/jaegertracing/migrating-from-jaeger-client-to-opentelemetry-sdk-bd337d796759))
* Linked to the spec for OpenCensus for now, not sure if this works in the long run and if we might need a separate migration guide like we have one for ot

### CODE REVIEW

**Review by cartermp — APPROVED** [review]
Awesome, love it!

**Comment by cartermp** [comment]
Looks like there's another formatting push to make here 🙃

File: content/en/docs/migration/_index.md [hunk 1]
```diff
@@ -3,3 +3,29 @@ title: Migration
 description: How to migrate to OpenTelemetry
 weight: 50
 ---
+
+## OpenTracing and OpenCensus
+
+OpenTelemetry was created as a merger of OpenTracing and OpenCensus. From the
+start OpenTelemetry was considered [to be the next major version of both
```
Comment (chalin) [inline content/en/docs/migration/_index.md:10]:
```suggestion
start, OpenTelemetry was considered [to be the next major version of both
```

File: content/en/docs/migration/_index.md [hunk 2]
```diff
@@ -3,3 +3,29 @@ title: Migration
...
+OpenTracing and OpenCensus][]. Because of that, one of the [key goals][] of the
+OpenTelemetry project is to provide backward compatibility with OpenCensus and a
```
Comment (chalin) [inline content/en/docs/migration/_index.md:12]: Why only mention OpenCensus here in terms of backwards compatibility?

File: content/en/docs/migration/_index.md [hunk 3]
```diff
@@ -3,3 +3,29 @@ title: Migration
...
+migration story for existing users.
+
+If you come from one of these projects, you can follow the migration guides for
+both [OpenTracing](./opentracing/) and [OpenCensus](./opencensus/)
```
Comment (chalin) [inline content/en/docs/migration/_index.md:16]:
```suggestion
both [OpenTracing](opentracing/) and [OpenCensus](opencensus/)
```

File: content/en/docs/migration/opencensus.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+title: Migrating from OpenCensus
+linkTitle: OpenCensus
+redirect: https://opentelemetry.io/docs/reference/specification/compatibility/opencensus/#migration-path
```
Comment (chalin) [inline content/en/docs/migration/opencensus.md:4]:
```suggestion
redirect: /docs/reference/specification/compatibility/opencensus/#migration-path
```

File: .htmltest.yml [hunk 5]
```diff
@@ -8,6 +8,7 @@ IgnoreURLs: # list of regexs of paths or URLs to be ignored
   - ^/docs/instrumentation/\w+/(api|examples)/$
   - ^/docs/instrumentation/net/(metrics-api|traces-api)/
   - ^/community/end-user/feedback-survey/$
+  - ^/docs/migration/opencensus/$
```
Comment (chalin) [inline .htmltest.yml:11]:
```suggestion
  - ^(/docs/migration/)?opencensus/$
```

**Review by chalin — COMMENTED** [review]
Very nice!

See inline comments for link-check fix and more.

**Comment by chalin** [comment]
> Looks like there's another formatting push to make here 🙃

@cartermp - the formatting seems file (the action is passing) -- unless I missed something. Were you referring to the link-check failure?

File: content/en/docs/migration/_index.md (hunk 2 above)
Comment (svrnm) [reply content/en/docs/migration/_index.md]: good call! fixed

**Review by chalin — APPROVED** [review]
👍

**Pull request merged**